frontend/src/entry/model/settings/TemplateSettings.tsx

```diff
@@ -0,0 +1,63 @@
+import { LoadingButton } from '@mui/lab'
+import { Checkbox, Divider, FormControlLabel, Stack, Typography } from '@mui/material'
+import { patchModel } from 'actions/model'
+import { useState } from 'react'
+import useNotification from 'src/hooks/useNotification'
+import MessageAlert from 'src/MessageAlert'
+import { EntryInterface } from 'types/types'
+import { getErrorMessage } from 'utils/fetcher'
+
+type TemplateSettingsProps = {
+  model: EntryInterface
+}
+
+export default function TemplateSettings({ model }: TemplateSettingsProps) {
+  const [allowTemplate, setAllowTemplate] = useState(model.settings.allowTemplating)
```

Comment:
Minor, but could this be renamed to `allowTemplating` and `setAllowTemplating` so that we don't have two slightly different names for the same property?